api/transport/invoker.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
maybe `handler_invoker.go` to make it more obvious that this is for invoking handlers